pkg/webhooks/handler/webhooks.go

```diff
@@ -28,6 +28,12 @@ const (
 	updateDryRunTimeOut = time.Second * 3
 )
 
+type WebhookHandlerIfs interface {
+	v1beta1.ValidatorWebhookHandler
+	Init(logger logr.Logger, cli client.Client, namespace string, isOpenshift bool)
+	HandleMutatingNsDelete(ns *corev1.Namespace, dryRun bool) (bool, error)
```

Comment:
@nunnatsa  we may think about moving this nsmutator into another interface in another package.